packages/insomnia/src/common/insomnia-v5.ts

```diff
@@ -489,17 +488,10 @@ export async function getInsomniaV5DataExport({
       throw new Error('Workspace not found');
     }
 
-    const workspaceDescendants = await database.withDescendants(
-      workspace,
-      null,
-      [],
-      WORKSPACE_EXPORT_TYPES_DESCENDANT_MAP,
-    );
-
-    const exportableTypes = Object.values(models.MODELS_BY_EXPORT_TYPE).map(model => model.type);
+    const workspaceDescendants = await database.getWithDescendants(workspace, models.EXPORTABLE_TYPES);
 
     const exportableResources = workspaceDescendants.filter(resource => {
-      if (exportableTypes.includes(resource.type)) {
+      if ((models.EXPORTABLE_TYPES as string[]).includes(resource.type)) {
```

Comment:
Nit: IMO, this would be more accurate.
```suggestion
      if (models.EXPORTABLE_TYPES.includes(resource.type as models.ModelTypes[number])) {
```